msu/hooks/ai/tactical/agent.nut

```diff
@@ -1,11 +1,8 @@
-::mods_hookBaseClass("ai/tactical/agent", function(o) {
-	o = o[o.SuperName];
+::MSU.HooksMod.hook("scripts/ai/tactical/agent", function(q) {
+	q.m.MSU_BehaviorStacks <- {};
```

Comment:
This probably needs to be initialized to `null` and then set to `{}` during a hook on create, right?

Reply:
yes

Reply:
Based on our discussion in Discord we found that this only applies to instances of classes. Arrays and Tables are properly cloned (perhaps raps did it on the backend with the embedded squirrel in BB). So we don't **need** to do this, but let's decide what we want to do.

Reply:
I personally prefer doing it explicitly, but if you and Taro disagree I'm ok with being outvoted here.